vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -88,6 +95,32 @@ public WebElement clearFieldLocatedByUsingKeyboard(SearchAttributes locator)
         return element;
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and check that it is
+     * sorted by text with using <b>sortingOrder</b> by element text.
+     * List should have more than 1 elements.
+     * @param locator to locate elements
```

Comment:
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@param 102 and 103